app.js

```diff
@@ -1,31 +1,168 @@
 'use strict';
 
-var picOne = newPicture();
+//working with Kayla and Caleb so our code may look similar
+//var picOne = newPicture();
+
+var imgContainer = document.getElementById('img-container');
+var leftEl = document.getElementById('left');
+var centerEl = document.getElementById('center');
+var rightEl = document.getElementById('right');
 
 var img = [
   './img/bag.jpg', './img/banana.jpg', './img/bathroom.jpg', './img/boots.jpg', './img/breakfast.jpg', './img/tauntaun.jpg', './img/wine-glass.jpg', './img/bubblegum.jpg', './img/chair.jpg', './img/cthulhu.jpg', './img/dog-duck.jpg', './img/dragon.jpg', './img/unicorn.jpg', './img/usb.gif', './img/pen.jpg', './img/pet-sweep.jpg', './img/scissors.jpg', './img/shark.jpg', './img/sweep.png', './img/water-can.jpg'];
 
-var previousImgs = [];
+var totalClicks = 0;
+var clickLimits = 25;
 
-function randomNum(){
-  return Math.floor(Math.floor(Math.random() * (20 - 1 + 1) + 1));
+function product(name){
+  this.name = name;
+  this.views = 0;
+  this.clicks = 0;
+  //this.path = 'img/' + name + jpg';
 };
 
+var previousImgs = [leftIndex, centerIndex, rightIndex];
+
+function randNum(){
+  return Math.floor(Math.floor(Math.random() * img.length));
+};
+
+/*function displayPics() {
+  var leftIndex = randomNum();
+  var centerIndex = randomNum();
+  var rightIndex = randomNum();
+}*/
+
 var allProducts = [];
-var leftIndex = randomNum();
+
+var leftIndex = randNum();
 var leftImg = allProducts[leftIndex];
 previousImgs.push(leftIndex);
 
-var centerIndex = randomNum();
+var centerIndex = randNum();
 while (centerIndex === leftIndex) {
-  centerIndex = randomNum();
+  centerIndex = randNum();
 }
 var centerImg = allProducts[centerIndex];
 previousImgs.push(centerIndex);
 
-var rightIndex = randomNum();
-while(rightIndex === leftIndex || rightIndex === centerIndex){
-  var rightIndex = randomNum();
+var rightIndex = randNum();
+while(rightIndex === leftIndex || rightIndex === centerIndex){ var rightIndex = randNum();
```

Comment:
Can you think of a way to rewrite these so that you're iterating through a loop instead? 

Also, make sure to break the stuff after the curly brace down onto the next line.